### PR TITLE
Issue #176: !comfirm confirms first, then deletes messages

### LINKED ISSUE #176: !confirm should confirm user first, then delete messages
To save time/confusion, the command should do the actions in this order

### PR DESCRIPTION
**Pull request by Nydauron**
As mentioned in title. Confirmation works as intended. Wasn't able to test deletion of unconfirmed users' messages. Closes #176.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1435,9 +1435,24 @@ async def pronouns(ctx, *args):
 @commands.check(isLauncher)
 async def confirm(ctx, *args: discord.Member):
     """Allows a staff member to confirm a user."""
+    msgs = []
     for i, member in enumerate(args):
```
Comment (cbrxyz) [inline bot.py:1439]: Why is this an `enumerate`? You aren't using i anywhere in the loop.

File: bot.py [hunk 2]
```diff
@@ -1435,9 +1435,24 @@ async def pronouns(ctx, *args):
...
+        role2 = discord.utils.get(member.guild.roles, name="Member")
+        await member.remove_roles(role1)
+        await member.add_roles(role2)
+        message = await ctx.send(f"Alrighty, confirmed {member.mention}. Welcome to the server! :tada:")
+        msgs.append(message)
+
+    await asyncio.sleep(3)
+    for i, m in enumerate(msgs):
+        if not i:
+            await ctx.message.delete()
```
Comment (cbrxyz) [inline bot.py:1452]: Why not bring `await ctx.message.delete()` out of the loop? It only needs to be called once.

File: bot.py [hunk 3]
```diff
@@ -1435,9 +1435,24 @@ async def pronouns(ctx, *args):
...
+        await m.delete()
+
+    for i, member in enumerate(args):
```
Comment (cbrxyz) [inline bot.py:1455]: Same here, what's the point of `enumerate`?

**Review by cbrxyz — COMMENTED** [review]
Left some comments throughout the code

**Comment by Nydauron** [comment]
Closing PR bc issue has been marked as resolved.

**Pull request closed without merging**